### ISSUE TITLE
Add documentation for 'contain' sorting

### ISSUE THREAD

**Issue by ConnorChristie** [cakephp/docs#3768]
So in the example below, there is no documentation about the sort keyword inside the contain block. I spent almost a half hour wondering why the order keyword wasn't working but then I saw that it is sort instead of order. It would be nice if you could document this in the docs because there is no talk about using the keyword 'sort' anywhere in the retrieving data section...
```
$extensionsTable
    ->find('all', [
        'contain' => [
            'Categories' => [
                'sort' => ['Categories.panel_row' => 'ASC', 'Categories.panel_column' => 'ASC'],
                'Settings' => [
                    'SettingValues'
                ]
            ]
        ]
    ]);
```

**Comment by markstory** [comment]
What version of CakePHP are you using? And where would you have expected to find this information?

**Comment by lorenzo** [comment]
that's 3.x

**Comment by ConnorChristie** [comment]
I would've expected to find it somewhere around this section: http://book.cakephp.org/3.0/en/orm/retrieving-data-and-resultsets.html#eager-loading-associations
But it does not talk about the options for a contain anywhere, especially not the sort option at least.

Thanks

**Comment by markstory** [comment]
Thanks @ConnorChristie I'll get the docs updated.

**Comment by markstory** [comment]
Fixed with #3769
